frontend/internal-packages/agent/src/langchain/agents/databaseSchemaBuildAgent/prompts.ts

```diff
@@ -7,71 +7,59 @@ Key responsibilities:
 - Confirm changes made
 - Suggest logical next steps
 
+VALIDATION REQUIREMENTS:
+- Tables must exist before adding columns to them
+- All required fields must be provided (see examples below)
+- Use exact JSON structure - no YAML syntax errors
+
 Use the schema manipulation tools to make changes and communicate clearly with users about what you're doing.
 
-Tool Usage Examples:
+Tool Usage Guidelines:
+
+CRITICAL: Always create tables before adding columns. Path format: /tables/{{table_name}}
+
+Required fields for ALL columns: name, type, notNull, default, comment, check
+Required fields for ALL tables: name, columns, comment, indexes, constraints
 
-Adding a new table:
+Minimal table example:
 {{
-  "operations": [
-    {{
-      "op": "add",
-      "path": "/tables/users",
-      "value": {{
-        "name": "users",
-        "columns": {{
-          "id": {{"name": "id", "type": "uuid", "notNull": true, "default": "gen_random_uuid()", "comment": "Unique identifier for each user", "check": null, "unique": false}},
-          "name": {{"name": "name", "type": "text", "notNull": true, "default": null, "comment": "Name of the user", "check": null, "unique": false}},
-          "email": {{"name": "email", "type": "text", "notNull": true, "default": null, "comment": "User email required for login", "check": null, "unique": true}}
-        }},
-        "comment": null,
-        "indexes": {{}},
-        "constraints": {{
-          "pk_users": {{
-            "type": "PRIMARY KEY",
-            "name": "pk_users",
-            "columnNames": ["id"]
-          }}
-        }}
+  "operations": [{{
+    "op": "add",
+    "path": "/tables/users",
+    "value": {{
+      "name": "users",
+      "columns": {{
+        "id": {{"name": "id", "type": "uuid", "notNull": true, "default": "gen_random_uuid()", "comment": "Primary key", "check": null}},
+        "email": {{"name": "email", "type": "text", "notNull": true, "default": null, "comment": "User email", "check": null}}
+      }},
+      "comment": null,
+      "indexes": {{}},
+      "constraints": {{
+        "pk_users": {{"type": "PRIMARY KEY", "name": "pk_users", "columnNames": ["id"]}}
       }}
     }}
-  ]
+  }}]
```

Comment:
The content remains the same, but the number of lines seems to have been compressed.
Thought this would be useful to make the context smaller